pkg/scheduler/algorithm/priorities/interpod_affinity.go

```diff
@@ -63,15 +64,15 @@ type podAffinityPriorityMap struct {
 	nodes []*v1.Node
 	// counts store the mapping from node name to so-far computed score of
 	// the node.
-	counts map[string]float64
+	counts map[string]*int64
```

Comment:
why does the value need to be an `int64` pointer, instead of just an `int64`?

Reply:
It's because [atomic#AddInt64](https://golang.org/pkg/sync/atomic/#AddInt64) takes pointer as parameter.

If we go with `map[string]int64`, map value is not addressable (`&map["key"]` is illegal); and alsoyou can't make `val := map["key"]; ptr := &val` b/c that's another address.

Reply:
That's right. In Go you cannot get a pointer to map entries as they may change during execution.